subjects/pingpong/README.md

```diff
@@ -0,0 +1,24 @@
+## pingpong
+
+### Instructions
+
+Write a program that checks if all numbers up to a given number are odd or even.
+The numbers will be between 0 and 100.
+
+- If the number is odd then it prints the number and `ping` as in the example below.
```

Comment:
In the usage you are saying that the number 2 for example is odd when 2 is an even number ...
This is wrong here you should be saying that 
"If the number is even then it prints the number and `ping` as in the example below."

Reply:
I'll correct that. Thank you.